fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/IDatabaseTranslator.java

```diff
@@ -50,6 +50,14 @@
      */
     boolean isDuplicate(SQLException x);
     
+    /**
+     * Check the exception to see if it is reporting that THE NAME OF THE OBJECT TO BE CREATED 
+     * OR THE TARGET OF A RENAME STATEMENT IS IDENTICAL TO THE EXISTING NAME OF THE OBJECT TYPE
+     * @param x
+     * @return
+     */
+    boolean isAlreadyExists(SQLException x);
```

Comment:
the name is a bit convoluted. 

isAlreadyExists  ... may be alternative 
alreadyExists
exists

Reply:
yeah, agree its a bit akward.  i went with `is` to match the other methods which are similar in nature.  its saying "is the SQLException an AlreadyExists one".

I had the same feeling about the existing `isDuplicate(SQLException x)` method name as well...

Reply:
I have no problemo with it.